lib/enigma.rb

```diff
@@ -31,7 +31,7 @@ def encrypt(message, key, date)
     encryption
   end
 
-  def decrypt(message, key, date)
+  def decrypt(message, key, date = current_date)
```

Comment:
Adds default value for date for decrypting. Key is still required as an argument for the decryption.